src/main/java/org/bookstore/service/impl/BookServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package org.bookstore.service.impl;
+
+import java.util.List;
+import org.bookstore.model.Book;
+import org.bookstore.repository.BookRepository;
+import org.bookstore.service.BookService;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
+
+    private final BookRepository bookRepository;
+
+    public BookServiceImpl(BookRepository bookRepository) {
+        this.bookRepository = bookRepository;
+    }
```

Comment:
Let’s  use the RequiredArgsConstraction annotation

Reply:
Changed